ui/app/src/utils/string.ts

```diff
@@ -157,3 +157,11 @@ export function getSimplifiedVersion(version: string, options: { minor?: boolean
   !minor && pieces.pop();
   return pieces.join('.');
 }
+
+export function unEscapeXml(value: string): string {
+  return value
+    .replace(/&lt;/g, '<')
+    .replace(/&gt;/g, '>')
+    .replace(/&quot;/g, '"')
+    .replace(/&amp;/g, '&');
+}
```

Comment:
Going forward, we're not going to be doing these transforms so it'd rather not encourage them by having this util (unless in the future we find a good reason for this). Let's do it inline in the tag value processor.